src/Controller/SentencesController.php

```diff
@@ -195,9 +195,8 @@ public function show($id = null)
             $tagsArray = $this->Sentences->getAllTagsOnSentence($id);
             $this->loadModel('SentencesSentencesLists');
             $listsArray = $this->SentencesSentencesLists->getListsForSentence($id);
-
+            
```

Comment:
You have some unnecessary spaces here.

(In order to detect such unnecessary whitespace changes, you could run `git diff` before `git add`ing or `git diff --staged` before `git commit`ting)